### PR TITLE
test- add theme and improve formatting

### PR DESCRIPTION
**Pull request by yyuanxuan**
This pull request includes some small tweaks to the formatting and a test of using Quarto theme from [Bootswatch](https://bootswatch.com/).

### CODE REVIEW

File: marking-criteria.qmd [hunk 1]
```diff
@@ -1,19 +1,20 @@
...
+  html:
+    theme: cerulean
 ---
 
 ## Marks
 
 Marks are awarded in 4 categories, accounting for the following criteria:
 
-    **Data processing: 20%**
+### Data processing: 20%
```
Comment (Robinlovelace) [inline marking-criteria.qmd:12]: Much better, was going to mention this, thank you!

File: marking-criteria.qmd [hunk 2]
```diff
@@ -1,19 +1,20 @@
-
 ---
 title: "Marking Criteria"
-format: html
+format:
+  html:
+    theme: cerulean
```
Comment (Robinlovelace) [inline marking-criteria.qmd:5]: Shouldn't we apply the them site-wide?

**Review by Robinlovelace — APPROVED** [review]

File: marking-criteria.qmd (hunk 1 above)
Comment (yyuanxuan) [reply marking-criteria.qmd:12]: Thanks for the quick review and approve :)

File: marking-criteria.qmd (hunk 2 above)
Comment (yyuanxuan) [reply marking-criteria.qmd:5]: Yes, it makes sense to stick to one theme site-wide. This is a test at the moment. I think we can first identify the theme we like best and then apply it site-wide. There are many to choose from: https://bootswatch.com

Comment (Robinlovelace) [reply marking-criteria.qmd:5]: Great plan.

Comment (Robinlovelace) [reply marking-criteria.qmd:5]: Marking criteria looking good here: https://itsleeds.github.io/TDStests/marking-criteria.html

**Pull request merged**